include/pgduckdb/pgduckdb_guc.h

```diff
@@ -8,7 +8,7 @@ extern bool duckdb_enable_external_access;
 extern bool duckdb_allow_unsigned_extensions;
 extern bool duckdb_autoinstall_known_extensions;
 extern bool duckdb_autoload_known_extensions;
-extern int duckdb_max_threads_per_postgres_scan;
+extern int max_workers_per_postgres_scan;
```

Comment:
Let's prefix this with `duckdb_` to not polute the global namespace with a fairly generic name.
```suggestion
extern int duckdb_max_workers_per_postgres_scan;
```